internal/services/monitor/monitor_activity_log_alert_resource.go

```diff
@@ -422,7 +426,7 @@ func resourceMonitorActivityLogAlertCreateUpdate(d *pluginsdk.ResourceData, meta
 
 	t := d.Get("tags").(map[string]interface{})
 	parameters := activitylogalertsapis.ActivityLogAlertResource{
-		Location: utils.String(azure.NormalizeLocation("Global")),
+		Location: &location,
```

Comment:
Let's change this to 
```suggestion
		Location: pointer.To(location.Normalize(d.Get("location").(string))),
```